src/proofofwork.py

```diff
@@ -305,8 +304,7 @@ def init():
             bmpow.restype = ctypes.c_ulonglong
             _doCPoW(2**63, "")
             logger.info("Successfully tested C PoW DLL (stdcall) %s", bitmsglib)
-        except:
-            logger.error("C PoW test fail.", exc_info=True)
+        except (ValueError, Exception) as e:
```

Comment:
The general exception handler needs to be separate, it needs to do something else. `ValueError` indicates that the DLL is there but doesn't accept the parameters. In that case, we should silently try different parameters. But if there is a general exception, it means the DLL may be missing, or corrupt, or some other issue, and it that case it doesn't make sense to test it with different parameters, but it makes sens to notify the user.